### ISSUE TITLE
SCSS to CSS to ExtractTextPlugin with fonts causes SyntaxError

### ISSUE THREAD

**Issue by rbellamy** [webpack/css-loader#378]
## TL;DR:

My use case that we have a SCSS file that overrides lumX defaults.

I initially thought this was an ExtractTextPlugin problem (https://github.com/webpack/extract-text-webpack-plugin/issues/292), but the module trace shows otherwise.

### Steps to reproduce

├── css-loader@0.26.0 
├── file-loader@0.9.0 
├── resolve-url-loader@1.6.0 
├── sass-loader@4.0.2 
├── url-loader@0.5.7 
└── webpack@1.13.3 

1. `main.scss` => `lumX-overrides.scss` => `lumX.scss` => `materialdesignicons.scss` => `materialdesignicons.woff2`
2. `css!resolve-url!sass` (with some tweaks)
3. webpack compile

### Expected results

Chunked CSS

### Actual results

So the `sass-loader` traverses the dependency tree to compile the SCSS to CSS, and then passes the combined CSS to ExtractTextPlugin, where I see the following SyntaxError.

```
Child extract-text-webpack-plugin:
    chunk    {0} extract-text-webpack-plugin-output-filename 793 kB [rendered]
        [0] ./~/css-loader?sourceMap&modules&importLoaders=2&localIdentName=[name]__[local]!./~/resolve-url-loader!./~/sass-loader?config=sassLoader!./src/styles/main.scss 792 kB {0} [built] [5 errors]
           factory:1ms building:16990ms = 16991ms
        [1] ./~/css-loader/lib/css-base.js 1.51 kB {0} [built]
            cjs require ./../../node_modules/css-loader/lib/css-base.js [0] ./~/css-loader?sourceMap&modules&importLoaders=2&localIdentName=[name]__[local]!./~/resolve-url-loader!./~/sass-loader?config=sassLoader!./src/styles/main.scss 1:27-85
           [0] 16991ms -> factory:14ms building:4ms = 17009ms
    
    ERROR in ./~/mdi/fonts/materialdesignicons-webfont.woff2
    Module parse failed: /home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/mdi/fonts/materialdesignicons-webfont.woff2 Unexpected character '' (1:4)
    You may need an appropriate loader to handle this file type.
    SyntaxError: Unexpected character '' (1:4)
        at Parser.pp$4.raise (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/node_modules/acorn/dist/acorn.js:2221:15)
        at Parser.pp$7.getTokenFromCode (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/node_modules/acorn/dist/acorn.js:2756:10)
        at Parser.pp$7.readToken (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/node_modules/acorn/dist/acorn.js:2477:17)
        at Parser.pp$7.nextToken (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/node_modules/acorn/dist/acorn.js:2468:15)
        at Parser.pp$7.next (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/node_modules/acorn/dist/acorn.js:2413:10)
        at Parser.pp$3.parseIdent (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/node_modules/acorn/dist/acorn.js:2191:10)
        at Parser.pp$3.parseExprAtom (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/node_modules/acorn/dist/acorn.js:1774:21)
        at Parser.pp$3.parseExprSubscripts (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/node_modules/acorn/dist/acorn.js:1715:21)
        at Parser.pp$3.parseMaybeUnary (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/node_modules/acorn/dist/acorn.js:1692:19)
        at Parser.pp$3.parseExprOps (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/node_modules/acorn/dist/acorn.js:1637:21)
        at Parser.pp$3.parseMaybeConditional (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/node_modules/acorn/dist/acorn.js:1620:21)
        at Parser.pp$3.parseMaybeAssign (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/node_modules/acorn/dist/acorn.js:1597:21)
        at Parser.pp$3.parseExpression (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/node_modules/acorn/dist/acorn.js:1573:21)
        at Parser.pp$1.parseStatement (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/node_modules/acorn/dist/acorn.js:727:47)
        at Parser.pp$1.parseTopLevel (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/node_modules/acorn/dist/acorn.js:638:25)
        at Parser.parse (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/node_modules/acorn/dist/acorn.js:516:17)
        at Object.parse (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/node_modules/acorn/dist/acorn.js:3098:39)
        at Parser.parse (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/lib/Parser.js:902:15)
        at NormalModule.<anonymous> (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack/lib/NormalModule.js:104:16)
        at NormalModule.onModuleBuild (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack-core/lib/NormalModuleMixin.js:310:10)
        at nextLoader (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack-core/lib/NormalModuleMixin.js:275:25)
        at /home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/webpack-core/lib/NormalModuleMixin.js:259:5
        at Storage.finished (/home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:38:16)
        at /home/rbellamy/Development/Terradatum/aergo/aergo-client/node_modules/graceful-fs/graceful-fs.js:78:16
        at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:415:3)
     @ ./~/css-loader?sourceMap&modules&importLoaders=2&localIdentName=[name]__[local]!./~/resolve-url-loader!./~/sass-loader?config=sassLoader!./src/styles/main.scss 6:287-360
```

### Notes
I've spent dozens of hours tweaking my loaders, the sass and css loader configurations, and trying to understand where things are going wrong. I've read every post, issue and article I can find about how to setup `sass-loader` and the ExtractTextPlugin to handle SCSS with font URLs, and I'm certain that's working - otherwise the `css-loader` wouldn't have even gotten to the point where it was trying to parse a font file.

_Question:_
How do I prevent the `css-loader` from trying to parse a font file referred to by SCSS?

## Detail
Here's my entire [webpack.config.js](https://gist.github.com/rbellamy/8b1240787efe132ee6d786c62b157244), but the relevant parts are:
*sassLoader*
```js
  sassLoader: {
    sourceMap: true,
    includePaths: [bourbon, path.resolve(__dirname, 'node_modules/mdi/scss')],
    excludePaths: [path.resolve(__dirname, 'node_modules')],
  }
```
*loaders*
```js
exports.scssprod = {
  test: /\.scss$/,
  loader: ExtractTextPlugin.extract('css?sourceMap&modules&importLoaders=2&localIdentName=[name]__[local]!resolve-url!sass?config=sassLoader'),
  exclude: /\.(eot|woff|woff2|ttf|svg)(\?[\s\S]+)?$/
};

exports.fonts = {
  test: /\.(eot|woff|woff2|ttf|svg)(\?[\s\S]+)?$/,
  loader: 'url-loader?limit=1000&name=fonts/[name].[ext]',
  exclude: /node_modules/,
  include: [path.resolve(__dirname, 'node_modules/mdi/fonts')]
};
```

**Comment by rbellamy** [comment]
I've been running webpack in the Chrome DevTools via `node --inspect`, and I can see the `css-loader` processing the SCSS file with the font urls and turning them into `require` statements with the proper paths to the font files.

I'm too much of a newbie to figure out where exactly things are going off the rails, but it's pretty clear to me that somewhere AFTER this point, right around where it tosses the ball to `css-base.js` or sometime immediately thereafter is when those require statements are parsed WITHOUT the benefit of the defined loader for those file types.

**Comment by rbellamy** [comment]
Oh. My. Gawd.

Should have been:
```
exports.fonts = {
  test: /\.(eot|woff|woff2|ttf|svg)(\?[\s\S]+)?$/,
  loader: 'url-loader?limit=1000&name=fonts/[name].[ext]',
  exclude: /node_modules/,
  include: [path.resolve(__dirname, '/node_modules/mdi/fonts')] // <== RIGHT HERE
};
```

I have to say that all of this could have been avoided if I'd been able to figure out how to see the computed configuration used by webpack... I didn't notice the path was missing the `/` until had it under the debugger.
```
